### PR TITLE
Update to Dune syntax

### PR DESCRIPTION
**Pull request by Leonidas-from-XIV**
I am not exactly sure what the proper behaviour of the module should be. I would think the `multipart` module should be just moved to `multipart_form_data` and be the only exposed module. Thus it would avoid wrapping, and have a nice mapping from package-name to module name.

### CODE REVIEW

File: lib/dune [hunk 1]
```diff
@@ -0,0 +1,5 @@
+(library
+  (name multipart_form_data)
+  (public_name multipart-form-data)
+  (libraries lwt stringext)
```
Comment (NathanReb) [inline lib/dune]: I'd rather keep the one lib per line form if you don't mind, this makes the diffs easier to read when it's updated!

Same applies to `pps` and in `tests/dune` of course.

**Review by NathanReb — COMMENTED** [review]
It's also probably worth an entry in the CHANGELOG.md since the toplevel module name changed at that makes it a breaking change.

Besides that it's :+1: was planning on updating anyway so tyvm!

File: test/tests.ml [hunk 2]
```diff
@@ -1,3 +1,5 @@
+module Multipart = Multipart_form_data
```
Comment (NathanReb) [inline test/tests.ml]: You can also probably just remove the module alias and use directly `Multipart_form_data` in the test.

**Comment by Leonidas-from-XIV** [comment]
Thanks @NathanReb, good to know what your position is on this. It is a good point to add a Changelog entry. I was also thinking to maybe remove `pkg/` altogether since `dune-release` does not need it anymore, so if you'd be willing to switch to that I can also remove it.

As an ex-Schemer and ex-Clojurist I find formatting lists this way strange, but it is your project so I also changed that 🙂

**Comment by NathanReb** [comment]
Sorry I was off for a couple weeks, I'll get back to this ASAP!

**Comment by NathanReb** [comment]
About removing `pkg/`, definitely in, maintaining `META` is a pain and I think `dune-release` fits perfectly for this project.

I you feel like removing it, be my guest, otherwise I can also do that later to get this merged.

**Comment by Leonidas-from-XIV** [comment]
I think it is better if you remove it, since I am on vacation now and access to my laptop is rather spotty :-)

**Comment by NathanReb** [comment]
Ok! Thanks for the contribution!

**Pull request merged**